### ISSUE TITLE
Gradle installDist task should install protoc + plugin if needed

### ISSUE THREAD

**Issue by chalin** [grpc/grpc-kotlin#6]
The gradle installDist task should have the necessary task dependencies to ensure that protoc and the Kotlin plugin get fetched and installed as necessary.

**Comment by marcoferrer** [comment]
I noticed the protoc plugin is also written in kotlin. Im not familiar with bazel but I am familiar with building a native executable for multiple archs with gradle and canteen. If interested I could open a PR with the proposed approach.

**Comment by chalin** [comment]
Thanks for the offer @marcoferrer, but @bshaffer is essentially done.

**Comment by chalin** [comment]
This is done, just waiting on #10.

**Comment by bshaffer** [comment]
This is no longer needed since we merged #7
